### PR TITLE
Fix: Remove foreign keys to token from simulation configuration

### LINKED ISSUE #406: Remove foreign keys to Token
## User stories
As a temporary fix for the missing foreign keys to from the component configuration to the token, remove the reference from simulation configuration to token.
That will make everything public to owners of a token.

## Acceptance Criteria
- [x] Remove reference from simulation configuration to token
- [x] Update docs
- [x] Update tests

### PR DESCRIPTION
**Pull request by antonykamp**
Fixes #<n>

Removes foreign key from simulation configuration to token in order to make the simulation configurations available for everyone with a token.

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [ ] Breaking changes anounced
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: tests/implementor/test_table_simulation_configuration.py [hunk 1]
```diff
@@ -17,28 +17,21 @@ def setup_method(self):
 
     @pytest.mark.parametrize(
         "init_dict",
-        [
-            {  # token does not exists
-                "token": "00000000-0000-0000-0000-000000000000",
-            },
-            {},  # token is missing
-        ],
+        [],
```
Comment (CR1337) [inline tests/implementor/test_table_simulation_configuration.py:20]:
```suggestion
        [{}],
```

File: tests/implementor/test_table_simulation_configuration.py [hunk 2]
```diff
@@ -17,28 +17,21 @@ def setup_method(self):
...
     )
     def test_create(self, init_dict: dict):
         """Test that an object of a class cannot be saved."""
         with pytest.raises(peewee.IntegrityError):
-            SimulationConfiguration(**init_dict).save(force_insert=True)
+            SimulationConfiguration(init_dict).save(force_insert=True)
```
Comment (CR1337) [inline tests/implementor/test_table_simulation_configuration.py]:
```suggestion
            SimulationConfiguration(**init_dict).save(force_insert=True)
```

File: tests/implementor/test_table_simulation_configuration.py [hunk 3]
```diff
@@ -17,28 +17,21 @@ def setup_method(self):
...
 
     @pytest.mark.parametrize(
         "init_dict",
-        [
-            {},  # token is missing
-        ],
+        [],
```
Comment (CR1337) [inline tests/implementor/test_table_simulation_configuration.py:29]:
```suggestion
        [{}],
```

File: tests/implementor/test_table_simulation_configuration.py [hunk 4]
```diff
@@ -17,28 +17,21 @@ def setup_method(self):
...
     )
     def test_deserialization(self, init_dict: dict):
         """Test that an object of a class cannot be deserialized."""
         with pytest.raises(marsh.exceptions.ValidationError):
-            SimulationConfiguration.Schema().load(init_dict)
+            SimulationConfiguration.Schema().load({**init_dict})
```
Comment (CR1337) [inline tests/implementor/test_table_simulation_configuration.py]:
```suggestion
            SimulationConfiguration.Schema().load(**init_dict)
```

**Review by CR1337 — CHANGES_REQUESTED** [review]
Looks good. I guess you did this to increase convenience during manual testing?
I suggested some changes to your test parametrization  to make it more inline with our other tests :)

**Comment by CR1337** [comment]
Also there seems to be a problem with Coveralls. The CI fails due to an internal server error :/

File: tests/implementor/test_table_simulation_configuration.py (hunk 1 above)
Comment (antonykamp) [reply tests/implementor/test_table_simulation_configuration.py:20]: Creating a SimulationConfiguration without any arguments is fine now :)

File: tests/implementor/test_table_simulation_configuration.py (hunk 3 above)
Comment (antonykamp) [reply tests/implementor/test_table_simulation_configuration.py:29]: Creating an SimulationConfiguration without any arguments is fine now :)

**Review by CR1337 — APPROVED** [review]
Ok, looks fine then. Good job :)

**Pull request merged**